sheetstorm/management/commands/data_load.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from users.factories import UserFactory
+
+CustomUser = get_user_model()
+
+admin_user_type = CustomUser.UserType.ADMIN.name
+employee_user_type = CustomUser.UserType.EMPLOYEE.name
+manager_user_type = CustomUser.UserType.MANAGER.name
+superuser_user_type = "SUPERUSER"
+
+
+class Command(BaseCommand):
+    help = "Create initial sample data for SheetStorm application testing."
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-a",
+            "--admin",
+            dest=admin_user_type,
+            type=int,
+            help="Indicates the maximum number of admins to be in the database",
+        )
+        parser.add_argument(
+            "-e",
+            "--employee",
+            dest=employee_user_type,
+            type=int,
+            help="Indicates the maximum number of employees to be in the database",
+        )
+        parser.add_argument(
+            "-m",
+            "--manager",
+            dest=manager_user_type,
+            type=int,
+            help="Indicates the maximum number of managers to be in the database",
+        )
+        parser.add_argument(
+            "-s",
+            "--superuser",
+            dest=superuser_user_type,
+            action="store_true",
+            help="Create and add superuser to the database if not exist",
+        )
+
+    @staticmethod
+    def add_user(user_type, number_of_users_to_create=1):
+        if user_type == admin_user_type:
+            factory_parameters = {"user_type": admin_user_type, "is_staff": True}
+
+        elif user_type == employee_user_type:
+            factory_parameters = {"user_type": employee_user_type}
+
+        elif user_type == manager_user_type:
+            factory_parameters = {"user_type": manager_user_type}
+
+        elif user_type == superuser_user_type:
+            factory_parameters = {"user_type": admin_user_type, "is_superuser": True, "is_staff": True}
+
+        for user_number in range(number_of_users_to_create):
+            user_email = f"user.{user_type}{user_number}@codepoets.it".lower()
+            user = CustomUser.objects.filter(email=user_email).first()
+            if user is None:
+                user = UserFactory(**factory_parameters, email=user_email)
+                user.save()
```

Comment:
Factories does not need to be saved because they are saved automatically when they are created. You can get rid of `user.save()` line so it would also simplify to:
```python
UserFactory(**factory_parameters, email=user_email)
```

Reply:
Done